python/doc/theory/reliability_sensitivity/sorm_approximation.rst

```diff
@@ -3,92 +3,96 @@
-SORM is to evaluate the probability content of the event
-:math:`\cD_f = \{\vect{X} \in \Rset^n \, / \, g(\vect{X}\,,\,\vect{d}) \le 0\}`
-:
+The Second Order Reliability Method is used under the following context: let :math:`\vect{X}` be a probabilistic input
+vector with joint density probability  :math:`\pdf`, let
+:math:`\vect{d}` be a  deterministic vector, let :math:`g(\vect{X}\,,\,\vect{d})` be the limit state function of
+the model and let :math:`\cD_f = \{\vect{X} \in \Rset^n \,
+/ \, g(\vect{X}\,,\,\vect{d}) \le 0\}` be an event which probability
+:math:`P_f` is defined as:
 
 .. math::
   :label: PfX4
 
-    P_f = \Prob{g(\vect{X}\,,\,\vect{d})\leq 0} = \int_{\cD_f}  \pdf\, d\vect{x}
+  P_f = \Prob{g(\vect{X}\,,\,\vect{d})\leq 0} = \int_{\cD_f}  \pdf\, d\vect{x}
 
-| The principle is the same as for FORM. After having mapped the
-  physical space into the standard through an isoprobabilistic
-  transformation :eq:`PfX4` becomes:
+The principle is the same as for :doc:`form_approximation`: we map the physical space into the standard through an isoprobabilistic transformation so that integral :eq:`PfX4` becomes:
 
-  .. math::
+.. math::
     :label: PfU2
 
-        P_f = \Prob{G(\vect{U}\,,\,\vect{d})\leq 0} = \int_{\Rset^n} \boldsymbol{1}_{G(\vect{u}\,,\,\vect{d}) \leq 0}\,f_{\vect{U}}(\vect{u})\,d\vect{u}
-
-  where :math:`f_{\vect{U}}` is the density function of the
-  distribution in the standard space : that distribution is spherical
-  (invariant by rotation by definition). That property implies that
-  :math:`f_{\vect{U}}` is a function of :math:`||\vect{U}||^2` only.
-  Furthermore, we suppose that outside the sphere which tangents the
-  limit state surface in the standard space, :math:`f_{\vect{U}}` is
-  decreasing.
-
-| The difference with FORM comes from the approximation of the limit
-  state surface at the design point :math:`\vect{P}^*` in the standard
-  space: SORM approximates it by a quadratic surface that has the same
-  main curvatures at the design point.
-| Let us denote by :math:`n` the dimension of the random vector
-  :math:`\vect{X}` and :math:`(\kappa_i)_{1 \leq i \leq n-1}` the
-  :math:`n-1` main curvatures of the limit state function at the design
-  point in the standard space.
-| Several approximations are available,
-  detailed here in the case where the origin of the standard
-  space does not belong to the failure domain :
-
--  Breitung’s formula is an asymptotic results: the
-   usual formula used in the normal standard space, has been generalized
-   in [lebrun2009b]_ to standard spaces where the
-   distribution is spherical, with :math:`E` the marginal cumulative
-   density function of the spherical distributions in the standard space:
+    P_f = \Prob{G(\vect{U}\,,\,\vect{d})\leq 0} = \int_{\Rset^n} \boldsymbol{1}_{G(\vect{u}\,,\,\vect{d}) \leq 0}\,f_{\vect{U}}(\vect{u})\,d\vect{u}
+
+where :math:`f_{\vect{U}}` is the density function of the   distribution in the standard space: that 
```

Comment:
This should fix the linter error:
`sorm_approximation.rst:25: D002 Trailing whitespace`
```suggestion
where :math:`f_{\vect{U}}` is the density function of the   distribution in the standard space: that
```

Reply:
Thanks for your comment... I have already seen it and corrected it in my latest commit...